### DISCUSSION TITLE
在ubantu编译通过，运行linux的demo出现 段错误

### DISCUSSION THREAD

**Discussion by wfd2022** [Tencent/libpag#2146] [Q&A]
在ubantu编译通过，运行linux的demo出现 段错误；用gdb调试如下
![企业微信截图_17083372825797](https://github.com/Tencent/libpag/assets/91302457/6f560200-0685-436a-bdda-6bfa11ff68fc)

**Comment by CodeJhF** [comment]
抱歉这条当时没有被跟进到，是我们漏了。

如果你还在使用 libpag 且问题仍然存在，麻烦回复一下当前版本号（4.4 / 4.5 系列）和最小复现 demo，我们会优先跟进；如果不再需要跟进，本条就先关闭了。（stale-close）
